src/compiler/crystal/codegen/debug.cr

```diff
@@ -221,6 +230,77 @@ module Crystal
       di_builder.create_array_type(type.size.as(NumberLiteral).value.to_i, llvm_typer.pointer_size, debug_type, [subrange])
     end
 
+    def create_debug_type(type : TypeDefType, type_name : String? = type.to_s)
+      # debug_compiler_log { "create_debug_type(#{type} : #{type.class}, type_name = #{type_name}, type.typedef = #{type.typedef}" }
```

Comment:
You might be able to remove this.

Reply:
I think I did. I thought I removed all debug info that not needed anymore. Will remove.

Reply:
Please also run `crystal tool format` to make CI happy.

Reply:
Ok debug logs are removed and code is formatted.